sekizai/helpers.py

```diff
@@ -34,6 +27,14 @@ def is_variable_extend_node(node):
             return True
     return False
 
+def get_context():
+    if engines is not None:
+        context = Context()
+        context.template = Template('')
+        return context
+    else:
+        return {}
```

Comment:
it'll make the API of this function cleaner, so I think changing it would be nice. also please update https://github.com/ojii/django-sekizai/blob/master/sekizai/helpers.pyi